iijpta/ngx_http_secure_token_iijpta.c

```diff
@@ -190,18 +197,38 @@ ngx_secure_token_iijpta_get_var(
 		goto error;
 	}
 	outp += out_len;
-
 	out_len = outp - out;
-	p = ngx_pnalloc(r->pool, sizeof("pta=") + (out_len * 2));
-	if (p == NULL)
+
+	if (conf->avoid_cookies == 0)
 	{
-		goto error;
+	        p = ngx_pnalloc(r->pool, sizeof("pta=") + (out_len * 2) +
```

Comment:
Add a new local var `size_t size`, initialize it to sizeof("pta=") + (out_len * 2), and if cookies are used, do += COOKIE_ATTR_SIZE